accounts/templates/partials/_read_book_list.html

```diff
@@ -0,0 +1,12 @@
+<div class="flex flex-col space-y-11 p-8 max-md:p-2 overflow-y-auto max-h-96" id="recommendation-list">
+    {% if goal.books_read.all %}
+    {% for book in goal.books_read.all %}
+    {% include 'partials/_read_book.html' with book=book %}
+    {% if not forloop.last %}
+    <hr />
+    {% endif %}
+    {% endfor %}
+    {% else %}
+    <p>No Books Read, yet. Keep Going!</p>
+    {% endif %}
```

Comment:
no indentations?

Reply:
brother, I wish the linter/formatter did that, but supposedly the proper formatting is with not indentations.